### PR TITLE
Add pom and resources to generate a basic PowSyBl distribution + update README + add github actions

### PR DESCRIPTION
**Pull request by MioRtia**
Signed-off-by: RALAMBOTIANA MIORA <miora.ralambotiana@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** 
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
First real commit

**What is the current behavior?** *(You can also link to an open issue here)*
This repository does nothing

**What is the new behavior (if this is a feature change)?**
From this repository, a basic PowSyBl distribution can be generated.

**Does this PR introduce a breaking change or deprecate an API?** 
No

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Maven projects
+/cgmes-validation/target/
+/distribution-validation/target/
```
Comment (agnesLeroy) [inline .gitignore:3]: I guess here we only have target to ignore

File: README.md [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+
+NB: Please note that this distribution **will not** take your personal `config.yml` file into account if you have one.
+
+To generate a PowSyBl basic distribution, you can then launch from the root repository:
+```
+$ cd ${PROJECT_ROOT_PATH}
+$ mvn clean package
+```
+
+The PowSyBl distribution is generated in the `${PROJECT_ROOT_PATH}/target` folder.
```
Comment (agnesLeroy) [inline README.md:58]: Here we could also recommend to add the path to target/powsybl/bin in the $PATH variable for an easier use (and also the equivalent on Windows).

File: .gitignore (hunk 1 above)
Comment (MioRtia) [reply .gitignore:3]: good call

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>package-zip</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <packageName>powsybl</packageName>
+                    <configName>config-distrib</configName>
```
Comment (mathbagu) [inline pom.xml:39]: It's not necessary to override the default parameters.

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>package-zip</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <packageName>powsybl</packageName>
```
Comment (mathbagu) [inline pom.xml:38]: Could be nice to include the version in the archive name.

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mathbagu) [inline pom.xml:1]: Add the license header

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-scripting</artifactId>
+            <version>${powsyblcore.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-security-analysis-api</artifactId>
+            <version>${powsyblcore.version}</version>
+        </dependency>
```
Comment (mathbagu) [inline pom.xml:59]: All dependencies are runtime dependencies?

File: README.md [hunk 7]
```diff
@@ -1,2 +1,64 @@
...
+</p>
+
+Read more at https://www.powsybl.org !
+
+This project and everyone participating in it is governed by the [PowSyBl Code of Conduct](https://github.com/powsybl/.github/blob/master/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior to [powsybl-tsc@lists.lfenergy.org](mailto:powsybl-tsc@lists.lfenergy.org).
+
+## PowSyBl vs PowSyBl Distribution
+
+PowSyBl Distribution allows the generation of a basic distribution of PowSyBl. It supports the following itools commands:
```
Comment (agnesLeroy) [inline README.md:25]: In this sentence it should probably be "allows for the generation"

**Pull request merged**